### DISCUSSION TITLE
how to get screen settings ?

### DISCUSSION THREAD

**Discussion by pbranly** [Blackymas/NSPanel_HA_Blueprint#788] [Q&A]
It is written in the doc that there should be a menu for adjusting light, timers , etc ..
I don,t find it in my screen
What is te setting ?

Thanks
Phil

**Comment by edwardtfn** [comment]
On the buttons pages, if you long press (+1sec, I think) any light, you will get to the light settings pages.

**Reply by pbranly** [reply]
Hi this is not what I need 
I need the way to change the screen light and dimmer 
I see that there is a specific horizontal screen but I cannot find the way to get it
Phil

**Reply by edwardtfn** [reply]
Do you mean this screen?
![](https://user-images.githubusercontent.com/41958506/203654386-e4e574ad-8674-4268-84dd-1c4e40f98eb9.png)

That one is available with a long press in the time displayed on the Home page.

**Reply by edwardtfn** [reply]
These are available from the long press in the button page:

![](https://user-images.githubusercontent.com/41958506/203654055-943d1910-7673-4d9f-ad81-7ef00d155e5a.png)
![](https://user-images.githubusercontent.com/41958506/203654076-93e110df-f314-4cf1-8500-ed667f2202fd.png)
![](https://user-images.githubusercontent.com/41958506/203654179-f7303b02-c886-4890-b976-cb498940a627.png)

**Reply by pbranly** [reply]
Thanks very much 
I will discover everyday !
Phil
